SDL/Triplet.h

```diff
@@ -16,10 +16,10 @@ namespace SDL {
     unsigned int* nMemoryLocations;
     uint8_t* logicalLayers;
     unsigned int* hitIndices;
-    //delta beta = betaIn - betaOut
-    FPX* betaIn;
-    FPX* betaOut;
-    FPX* pt_beta;
+    float* betaIn;
```

Comment:
```suggestion
    FPX* betaIn;
```
betaIn should be OK in half-precision (same comment to the buffer definition)